_inc/lib/core-api/wpcom-endpoints/memberships.php

```diff
@@ -143,19 +152,22 @@ public function create_product( $request ) {
 	/**
 	 * Get a status of connection for the site. If this is Jetpack, pass the request to wpcom.
 	 *
+	 * @param object $request - request passed from WP.
```

Comment:
Maybe we can be more explicit here?

```suggestion
	 * @param \WP_REST_Request $request - request passed from WP.
```

Reply:
Oh sure, thanks!